### PR TITLE
fix: nft wrapped fees / no wrap fees

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)
If wrapped fees are needed for make offer, show the row.  Otherwise hide it and don't include in total.

Screen Shot 2022-06-22 at 9 47 16 AM

Screen Shot 2022-06-22 at 9 47 21 AM

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/fees.tsx [hunk 1]
```diff
@@ -11,16 +11,19 @@ import CreateOfferFees from './CreateOffer.fees'
...
 
 const Fees: React.FC<Props> = (props) => {
-  const { isAuthenticated, isInvited } = props
+  const { canWrap, isAuthenticated, isInvited, needsWrap } = props
 
   const getTotalFees = () => {
-    const totalFees = new BigNumber(props?.orderFlow?.wrapEthFees?.data?.approvalFees)
-      .multipliedBy(props?.orderFlow?.wrapEthFees?.data?.gasPrice)
-      .plus(
+    let totalFees = new BigNumber(props?.orderFlow?.wrapEthFees?.data?.approvalFees).multipliedBy(
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/fees.tsx:17]: why not get orderFlow from props destructuring from above?

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/WrapEth.fees.tsx [hunk 2]
```diff
@@ -53,10 +53,14 @@ const Fees: React.FC<Props> = (props) => {
               <div>
                 <RightAlign>
                   <CoinDisplay size='14px' color='black' weight={600} coin='ETH'>
-                    {new BigNumber(val.totalFees).multipliedBy(val.gasPrice).toString()}
+                    {canWrap && needsWrap
+                      ? new BigNumber(val.totalFees).multipliedBy(val.gasPrice).toString()
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/WrapEth.fees.tsx:57]: `val` could also be using destructuring

Comment (julianromero-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/WrapEth.fees.tsx:57]: Also it seems that` new BigNumber(val.totalFees).multipliedBy(val.gasPrice).toString()` is duplicated several times, you could simplify creating a constant an using it each time

**Review by julianromero-bc — APPROVED** [review]
Minor comments but it looks fine

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/WrapEth.fees.tsx (hunk 2 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/WrapEth.fees.tsx:57]: lets leave for now, thanks!

**Pull request merged**